content/news/news.20241219.md

```diff
@@ -0,0 +1,8 @@
+---
+title: "Taskwarrior 3.3.0 Released"
+date: 2024-12-19
+---
+
+### Taskwarrior 3.3 Released
```

Comment:
`3.3.0`?

I would use the whole version string, also in the rest of the post. 

To me, the abbreviated form refers to *all* patch versions `3.3.x`, so it is not a specific release...